app/controllers/api_controller/collection_config.rb

```diff
@@ -0,0 +1,110 @@
+class ApiController
+  class CollectionConfig < Config::Options
```

Comment:
I'm wondering if it would be better to use composition over inheritance here. Are we using any `Config::Options` generated methods outside this class?